package-lock.json

```diff
@@ -1290,8 +1290,7 @@
     },
     "node_modules/brave-site-specific-scripts": {
       "version": "1.0.0",
-      "resolved": "git+https://git@github.com/brave/brave-site-specific-scripts.git#6f59813b833104234caa0f7ed2a0926526cc96ab",
-      "integrity": "sha512-wA+kog/dVR0Q/KTd1KP9sgxsSK130JjLwUJawUQyPk4CjHCbHuj2zzUL+3s+xTgybLGMWBO7gQyM39P3ITQ+lQ==",
```

Comment:
I wonder why the `integrity` bit is being dropped here. What version of npm are you using? `8.11.0` is the latest LTS.

Reply:
I'm using `8.5.0` on Linux. I could try updating if you think it will help. Both changes were a result of running `npm update brave-site-specific-scripts`.

Reply:
Per the comment in https://github.com/npm/npm/issues/21015#issuecomment-401166253, it seems that Git dependencies no longer use integrity hashes in newer npm versions.

Reply:
Ah interesting, so it's an unused leftover then.